cmd/utils.go

```diff
@@ -31,7 +31,12 @@ func prepareDefaultAirflowImageTag(airflowVersion string, httpClient *airflowver
 			defaultImageTag = fmt.Sprintf("%s-buster-onbuild", airflowVersion)
 		}
 	} else if airflowVersion != "" {
-		return "", errors.New("An Error occurred: Are you authenticated? If not - you can't use the --airflow-version option")
+		switch t := err; t {
+		default:
+			return "", err
```

Comment:
@paolaperaza  This will surface all not auth related errors with a message that looks similar to "API error (400): $THE_ERROR_MESSAGE_FROM_HOUSTON". This will be useful for debugging. But wanted to check to make sure we want to expose this. Alternatively we  could  mask the error with something generic E.G. An unexpected error occurred reach out to customer support.

Reply:
@aliotta Got it, that makes sense. We just talked about it over Zoom, but I actually think that as nice as "Talk to Support!" might sound, I'm worried we'd be masking the root cause error and make it even harder for our CRE team to solve it anyway.

Let's keep these errors raw for now, and then ideally if any of them are particularly common/persist then we can create messaging that addresses each/some of them individually.